dask_cuda/utils_test.py

```diff
@@ -0,0 +1,28 @@
+import os
+from zict.file import _safe_key as safe_key
+import string
+from random import choice
+
+
+def assert_device_host_file_size(dhf, total_bytes, chunk_overhead=1024):
+    byte_sum = dhf.device.total_weight + dhf.host.total_weight
+    file_path = [
+        os.path.join(dhf.disk.d.directory, safe_key(k))
+        for k in dhf.disk.keys()
+    ]
+    file_size = [os.path.getsize(f) for f in file_path]
+    byte_sum += sum(file_size)
+    print(file_size)
```

Comment:
Errant print statement?

Reply:
What print statement? You saw nothing. :D